python/lsst/shapelet/shapeletFunction/shapeletFunctionContinued.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, division, print_function
+
+from .shapeletFunction import ShapeletFunction
+
+from lsst.utils import continueClass
+
+__all__ = []
+
+@continueClass
+class ShapeletFunction:
+    def __reduce__(self):
+        return (ShapeletFunction, (self.getOrder(), self.getBasisType(),
+            self.getEllipse(), self.getCoefficients()))
```

Comment:
It is possible. But probably not worth it for now. We eventually need some utility code for that I think.